containers/agent/deploy.go

```diff
@@ -135,8 +135,10 @@ func CopyFromPod(kclient kubernetes.Clientset, pod corev1.Pod, containerName str
 	reader, writer := io.Pipe()
 
 	go func() {
-		defer writer.Close()
-		err := exec.Stream(remotecommand.StreamOptions{
+		defer func() {
+			_ = writer.Close()
+		}()
```

Comment:
Was this done so that any incidental error returned from `writer.Close()` is just ignored?